tests/topotests/lib/pim.py

```diff
@@ -466,76 +466,6 @@ def find_rp_details(tgen, topo):
     return rp_details
 
 
-def configure_pim_force_expire(tgen, topo, input_dict, build=False):
```

Comment:
Why is this being deleted?

Reply:
It is resolved, changes are merged via different PR.